[Bug] Value null at creationOffset of type org.json.JSONObject$1 cannot be converted to int

```
com.ichi2.utils.JSONException: org.json.JSONException: Value null at creationOffset of type org.json.JSONObject$1 cannot be converted to int
at com.ichi2.utils.JSONObject.getInt(JSONObject.java:2)
at com.ichi2.libanki.Config.getInt(Config.kt:1)
at com.ichi2.libanki.Collection.get_config(Collection.java:6)
at com.ichi2.libanki.sched.SchedV2._creation_timezone_offset(SchedV2.java:1)
at com.ichi2.libanki.sched.SchedV2._timingToday(SchedV2.java:3)
at com.ichi2.libanki.sched.SchedV2._updateCutoff(SchedV2.java:2)
at com.ichi2.libanki.sched.SchedV2.<init>(SchedV2.java:18)
at com.ichi2.libanki.Collection._loadScheduler(Collection.java:3)
at com.ichi2.libanki.Collection.<init>(Collection.java:20)
at com.ichi2.libanki.Storage.Collection(Storage.java:10)
at com.ichi2.anki.CollectionHelper.getCol(CollectionHelper.java:8)
at com.ichi2.anki.CollectionHelper.getCol(CollectionHelper.java:3)
at com.ichi2.anki.CollectionHelper.getColSafe(CollectionHelper.java:1)
at com.ichi2.anki.DeckPicker.onPrepareOptionsMenu(DeckPicker.java:1)
at android.app.Activity.onPreparePanel(Activity.java:4325)
at androidx.fragment.app.FragmentActivity.onPrepareOptionsPanel(FragmentActivity.java:1)
at androidx.fragment.app.FragmentActivity.onPreparePanel(FragmentActivity.java:1)
at androidx.appcompat.view.WindowCallbackWrapper.onPreparePanel(WindowCallbackWrapper.java:1)
at androidx.appcompat.app.AppCompatDelegateImpl$AppCompatWindowCallback.onPreparePanel(AppCompatDelegateImpl.java:3)
at androidx.appcompat.view.WindowCallbackWrapper.onPreparePanel(WindowCallbackWrapper.java:1)
at androidx.appcompat.app.ToolbarActionBar$ToolbarCallbackWrapper.onPreparePanel(ToolbarActionBar.java:1)
at androidx.appcompat.app.ToolbarActionBar.populateOptionsMenu(ToolbarActionBar.java:6)
at androidx.appcompat.app.ToolbarActionBar$1.run(ToolbarActionBar.java:1)
at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1076)
at android.view.Choreographer.doCallbacks(Choreographer.java:897)
at android.view.Choreographer.doFrame(Choreographer.java:822)
at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1061)
at android.os.Handler.handleCallback(Handler.java:938)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:236)
at android.app.ActivityThread.main(ActivityThread.java:8061)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:656)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:967)
Caused by: org.json.JSONException: Value null at creationOffset of type org.json.JSONObject$1 cannot be converted to int
at org.json.JSON.typeMismatch(JSON.java:101)
at org.json.JSONObject.getInt(JSONObject.java:492)
at com.ichi2.utils.JSONObject.getInt(JSONObject.java:1)
... 33 more
at androidx.appcompat.view.WindowCallbackWrapper.onPreparePanel(WindowCallbackWrapper.java:1)
```

--- a/AnkiDroid/src/main/java/com/ichi2/libanki/Collection.java
+++ b/AnkiDroid/src/main/java/com/ichi2/libanki/Collection.java
@@ -2143,7 +2143,7 @@ public class Collection implements CollectionGetter {
     @Nullable
     @Contract("_, !null -> !null")
     public Boolean get_config(@NonNull String key, @Nullable Boolean defaultValue) {
-        if (!mConf.has(key)) {
+        if (mConf.isNull(key)) {
             return defaultValue;
         }
         return mConf.getBoolean(key);
@@ -2152,7 +2152,7 @@ public class Collection implements CollectionGetter {
     @Nullable
     @Contract("_, !null -> !null")
     public Long get_config(@NonNull String key, @Nullable Long defaultValue) {
-        if (!mConf.has(key)) {
+        if (mConf.isNull(key)) {
             return defaultValue;
         }
         return mConf.getLong(key);
@@ -2161,7 +2161,7 @@ public class Collection implements CollectionGetter {
     @Nullable
     @Contract("_, !null -> !null")
     public Integer get_config(@NonNull String key, @Nullable Integer defaultValue) {
-        if (!mConf.has(key)) {
+        if (mConf.isNull(key)) {
             return defaultValue;
         }
         return mConf.getInt(key);
@@ -2169,7 +2169,7 @@ public class Collection implements CollectionGetter {
 
     @Contract("_, !null -> !null")
     public Double get_config(@NonNull String key, @Nullable Double defaultValue) {
-        if (!mConf.has(key)) {
+        if (mConf.isNull(key)) {
             return defaultValue;
         }
         return mConf.getDouble(key);
@@ -2178,7 +2178,7 @@ public class Collection implements CollectionGetter {
     @Nullable
     @Contract("_, !null -> !null")
     public String get_config(@NonNull String key, @Nullable String defaultValue) {
-        if (!mConf.has(key)) {
+        if (mConf.isNull(key)) {
             return defaultValue;
         }
         return mConf.getString(key);
@@ -2188,7 +2188,7 @@ public class Collection implements CollectionGetter {
     @Nullable
     @Contract("_, !null -> !null")
     public JSONObject get_config(@NonNull String key, @Nullable JSONObject defaultValue) {
-        if (!mConf.has(key)) {
+        if (mConf.isNull(key)) {
             return defaultValue == null ? null : new JSONObject(defaultValue);
         }
         return new JSONObject(mConf.getJSONObject(key));
@@ -2198,7 +2198,7 @@ public class Collection implements CollectionGetter {
     @Nullable
     @Contract("_, !null -> !null")
     public JSONArray get_config(@NonNull String key, @Nullable JSONArray defaultValue) {
-        if (!mConf.has(key)) {
+        if (mConf.isNull(key)) {
             return defaultValue == null ? null : new JSONArray(defaultValue);
         }
         return new JSONArray(mConf.getJSONArray(key));

--- a/AnkiDroid/src/test/java/com/ichi2/libanki/ConfigTest.kt
+++ b/AnkiDroid/src/test/java/com/ichi2/libanki/ConfigTest.kt
@@ -55,4 +55,15 @@ class ConfigTest : RobolectricTest() {
 
         assertThat("key removed", col.has_config_not_null("aa"), equalTo(false))
     }
+
+    @Test
+    fun get_config_uses_default() {
+        assertThat(col.get_config("hello", 1L), equalTo(1L))
+
+        val json = col.conf
+        json.put("hello", JSONObject.NULL)
+        col.conf = json
+
+        assertThat(col.get_config("hello", 1L), equalTo(1L))
+    }
 }

